Frameworks/CoreText/CTFrame.mm

```diff
@@ -108,12 +104,7 @@ void CTFrameGetLineOrigins(CTFrameRef frameRef, CFRange range, CGPoint origins[]
     }
 }
 
-/**
- @Status Interoperable
-*/
-void CTFrameDraw(CTFrameRef frame, CGContextRef ctx) {
-    uint32_t count = [((_CTFrame*)frame)->_lines count];
-    CGPoint curTextPos = CGContextGetTextPosition(ctx);
+void __CTFrameDraw(CTFrameRef frame, CGContextRef ctx) {
```

Comment:
any reason to have this helper anymore? just have this be the body of CTFrameDraw()